server/src/test/java/io/crate/integrationtests/JoinIntegrationTest.java

```diff
@@ -1726,7 +1726,19 @@ public void test_nested_left_outer_hash_join() throws Exception {
         execute("insert into t3 values (4), (5)");
         execute("refresh table t1, t2, t3");
 
-        execute("SELECT * FROM t1 LEFT JOIN (t2 LEFT JOIN t3 ON t2.id = t3.id) ON t2.id = t1.id;");
+        String query = "SELECT * FROM t1 LEFT JOIN (t2 LEFT JOIN t3 ON t2.id = t3.id) ON t1.id = t2.id";
+
+        execute("explain " + query);
+
+        assertThat(response).hasLines(
+            "HashJoin[LEFT | (id = id)] (rows=unknown)",
+            "  ├ Collect[doc.t1 | [id] | true] (rows=unknown)",
+            "  └ HashJoin[LEFT | (id = id)] (rows=unknown)",
+            "    ├ Collect[doc.t2 | [id] | true] (rows=unknown)",
+            "    └ Collect[doc.t3 | [id] | true] (rows=unknown)"
+        );
+
```

Comment:
Do we need this change given that we have a dedicated test?

Looks like difference is that here we use LEFT join and in the new test we use RIGHT join but that shouldn't be a fundamental difference?

I guess plan here is for correct and not related to this change? 
Otherwise result set assertion would have broken.

Reply:
So plan was incorrect on master but (accidentally) produced a correct result?

If so, fine to keep it

Reply:
But yes, i should have stated that. This gave me a lot of headache first, while this broke constantly while working on this issue.